pkg/services/ngalert/models/alert_rule.go

```diff
@@ -356,6 +356,11 @@ type AlertRuleKeyWithVersionAndPauseStatus struct {
 	AlertRuleKeyWithVersion `xorm:"extends"`
 }
 
+type AlertRuleKeyWithId struct {
+	AlertRuleKey
+	ID int64 `xorm:"id"`
```

Comment:
NIT: Is the xorm tag necessary?

Reply:
No. We specify tags in AlertRuleKey, so i decided to add it for consistency.

Reply:
I think the base model has the tags because the other structs are written to the db. If we don't plan on persisting this I'd say let's remove it. Wdyt?

Reply:
Agree. Removed the tag.